lib/container.js

```diff
@@ -5,6 +5,9 @@ const MochaFactory = require('./mochaFactory');
 const recorder = require('./recorder');
 const event = require('./event');
 const glob = require('glob');
+const { isMainThread, parentPort } = require('worker_threads');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`worker_threads` import should occur before import of `./utils`](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4759786)